### ISSUE TITLE
Using MSE loss for class prediction

### ISSUE THREAD

**Issue by irum** [YunYang1994/tensorflow-yolov3#606]
@YunYang1994  I am using probabilities for class prediction rather than class label . For example 
X,y,W,h, 10 probabilities x,y,W,h,10 probabilities....
In the text file. For this I have used MSE loss function but Yolov3 does not predict classes properly . Can you please give me your suggestion
